airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -654,21 +654,28 @@ class TicketMetrics(SourceZendeskSupportCursorPaginationStream):
     """TicketMetric stream: https://developer.zendesk.com/api-reference/ticketing/tickets/ticket_metrics/"""
 
     def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
-        if self._ignore_pagination:
-            return None
-        next_page = self._parse_next_page_number(response)
-        return next_page if next_page else None
+        """
+        https://developer.zendesk.com/documentation/api-basics/pagination/paginating-through-lists-using-cursor-pagination/#when-to-stop-paginating
+        """
+        meta = response.json().get("meta", {})
+        return meta.get("after_cursor") if meta.get("has_more", False) else None
```

Comment:
Hey @bazarnov, there is some discussion about this line in https://github.com/airbytehq/airbyte/pull/27070#discussion_r1228497844 and I am hoping you could help us understand. The contributor thinks it should be:

```suggestion
        return meta.get("after_cursor") if meta.get("has_more", True) else None
``` 

From what I can tell their logic seems correct, but I couldn't reproduce any bugs with your version and I don't want to change something that works.

What do you think about this? is there some context we're missing? any thoughts would be appreciated!

Reply:
Nevermind, figured it out that `False` is just default here 😅  Thank you!